src/lib/src/charts/timeseries-chart.component.ts

```diff
@@ -88,24 +97,28 @@ export class TimeseriesChartComponent implements OnChanges {
       }
 
       if (this.formatX) {
-        config.data['xFormat'] = this.formatX;
```

Comment:
TypeScript can be helpful at times. I found the typing for this config, and it turns out I was using `xFormat` incorrectly. It's supposed to be a string to tell C3 how to interpret the date formats, but since we're using standard ISO 8601, we don't really need it.